### PR TITLE
Fixes #2501: Replace CNTT with Anuket

### LINKED ISSUE #2501: [General] Replace CNTT with Anuket

### PR DESCRIPTION
**Pull request by msidana**

### CODE REVIEW

**Review by walterkozlowski — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ Any Pull Requests that are labeled as "**Unresolved**" will be discussed during
 
 
 <!--
-* [Contribution Guidelines](https://github.com/cntt-n/CNTT/wiki/Contribution-Guidelines)
-* [Approval Process](https://github.com/cntt-n/CNTT/wiki/Approval-Process) 
+* [Contribution Guidelines](https://github.com/cntt-n/Anuket CNTT/wiki/Contribution-Guidelines)
```
Comment (pgoyal01) [inline CONTRIBUTING.md:112]: You can't do this -- there is no such link

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ Any Pull Requests that are labeled as "**Unresolved**" will be discussed during
...
+* [Approval Process](https://github.com/cntt-n/Anuket CNTT/wiki/Approval-Process)
```
Comment (pgoyal01) [inline CONTRIBUTING.md:113]: You can't do this -- there is no such link

File: .github/ISSUE_TEMPLATE/cntt-issue.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-name: CNTT Issue
+name: Anuket CNTT Issue
```
Comment (pgoyal01) [inline .github/ISSUE_TEMPLATE/cntt-issue.md:2]: Why are we referring to "Anuket CNTT" as there is no such entity. APplies to many other changes.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -11,7 +11,7 @@
 
 <a name="permissions"></a>
 ### Permission levels
-There are different permissions (levels) available for contributing into CNTT:
+There are different permissions (levels) available for contributing into Anuket CNTT:
```
Comment (pgoyal01) [inline CONTRIBUTING.md:14]:
```suggestion
There are different permissions (levels) available for contributing to Anuket:
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ There are different permissions (levels) available for contributing into CNTT:
 
 <a name="labels"></a>
 ### Labels
-Every Issue and PR should be tagged with one or more of the following labels to indicate their status and for the automation pipelines to apply the correct CNTT processes.
+Every Issue and PR should be tagged with one or more of the following labels to indicate their status and for the automation pipelines to apply the correct Anuket CNTT processes.
```
Comment (pgoyal01) [inline CONTRIBUTING.md:25]:
```suggestion
Every Issue and Patch Request (PR) should be tagged with one or more of the following labels to indicate their status and for the automation pipelines to apply the correct Anuket processes.
```

File: README.md [hunk 6]
```diff
@@ -8,25 +8,25 @@
...
   and [the test case descriptions and selection](doc/ref_cert/RC2/chapters/chapter02.md)
   for more details.
 * **deploy OpenStack to onboard your VNFs?**
   You would be primarily interested by
-  [the CNTT reference implementation](doc/ref_impl/cntt-ri/chapters/chapter01.md)
+  [the Anuket CNTT reference implementation](doc/ref_impl/cntt-ri/chapters/chapter01.md)
 
 ## Term of Reference & License
-* [Term of Reference](GSMA_CNTT_Terms_of_Reference.pdf)
+* [Term of Reference](GSMA_Anuket CNTT_Terms_of_Reference.pdf)
```
Comment (pgoyal01) [inline README.md:26]:
```suggestion
* [Term of Reference](GSMA_CNTT_Terms_of_Reference.pdf)
```

File: README.md [hunk 7]
```diff
@@ -8,25 +8,25 @@
...
 * [Code of Conduct](CODE_OF_CONDUCT.md)
 * [The License (Creative Commons Attribution 4.0 International)](https://creativecommons.org/licenses/by/4.0/legalcode)
 
 ### Read the Docs:
-* [Wiki](https://github.com/cntt-n/CNTT/wiki)
-* [Doc](https://cntt-n.github.io/CNTT/)
+* [Wiki](https://github.com/cntt-n/Anuket CNTT/wiki)
```
Comment (pgoyal01) [inline README.md:31]:
```suggestion
* [Wiki](https://github.com/cntt-n/CNTT/wiki)
```

File: README.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# Cloud iNfrastructure Telco Task Force (CNTT)
+# Cloud iNfrastructure Telco Task Force (Anuket CNTT)
```
Comment (pgoyal01) [inline README.md:1]:
```suggestion
# Cloud iNfrastructure Telco Task Force (CNTT)
```

File: doc/README.md [hunk 9]
```diff
@@ -1,8 +1,8 @@
-# Cloud iNfrastructure Telco Task Force (CNTT)
+# Cloud iNfrastructure Telco Task Force (Anuket CNTT)
 
 ## Table of Contents
 * [Governance & Community Guidelines](gov/README.md)
-  * [Terms of Reference](../GSMA_CNTT_Terms_of_Reference.pdf)
+  * [Terms of Reference](../GSMA_Anuket CNTT_Terms_of_Reference.pdf)
```
Comment (pgoyal01) [inline doc/README.md:5]:
```suggestion
  * [Terms of Reference](../GSMA_CNTT_Terms_of_Reference.pdf)
```

**Review by pgoyal01 — CHANGES_REQUESTED** [review]
@msidana 
1. You can't change URLs or file names by prefixing "CNTT" with "Anuket ". They will result in errors. I have pointed this out in some places but the number of such changes is quite large.
1. Except where CNTT is defined or is relevant from a historical perspective you should replace CNTT with Anuket but not Anuket CNTT -- CNTT doesn't exist anymore.

@walterkozlowski This should not be approved at least until all url's and file names are restored.

**Comment by msidana** [comment]
> @msidana
> 
> 1. You can't change URLs or file names by prefixing "CNTT" with "Anuket ". They will result in errors. I have pointed this out in some places but the number of such changes is quite large.
> 2. Except where CNTT is defined or is relevant from a historical perspective you should replace CNTT with Anuket but not Anuket CNTT -- CNTT doesn't exist anymore.
> 
> @walterkozlowski This should not be approved at least until all url's and file names are restored.

@pgoyal01 I had not done the change at a few places to avoid breaking links/urls but some still made way due to overall replacement. I'll fix this to ensure there are no broken URLs. Within the repo, I have addressed this by changing filenames so that the linking doesn't breaks. Will update the PR and push again.

**Comment by CsatariGergely** [comment]
@msidana thanks for the pr, but you can not just replace "CNTT" with "Anuket" in every file. Please create a pr per document, like RM, RA1, RA2 and so on and "Anuket CNTT" is not a think, also if you replace CNTT to anything else in the absuolte links the links will break (instead please replace the absolulte links to relative ones ). I would propose to close this pr and open specific pr-s to the different document. @EditKoselak is also working on the Annuket-ization so please align with her on what documents you would like to address.

**Pull request closed without merging**